tools/reviewer/service/tests/CodeReviewServiceTest.java

```diff
@@ -158,10 +159,20 @@ private String getWorkspaceFolder(String workspace) {
     return joinPaths(aaBaseFolder, "ws", workspace);
   }
 
-  /**
-   * To test the server, make calls with a real stub using the in-process channel, and verify
-   * behaviors or state changes from the client side.
-   */
+  TextDiffResponse getResponse(File file) {
```

Comment:
- TextDiffResponse getResponse(File file);
- TextDiffResponse getExpectedResponse(String contents);
- void writeFile(String contents);
- void writeFile(String filename, String contents);
- void deleteFile(String filename);

Can we change access level to private?